packages/syft/src/syft/node/credentials.py

```diff
@@ -22,9 +22,10 @@
 class SyftVerifyKey(SyftBaseModel):
     verify_key: VerifyKey
 
-    @pydantic.validator("verify_key", pre=True, always=True)
-    def make_verify_key(cls, v: Union[str, VerifyKey]) -> VerifyKey:
-        return VerifyKey(bytes.fromhex(v)) if isinstance(v, str) else v
+    def __init__(self, verify_key: Union[str, VerifyKey]):
+        if isinstance(verify_key, str):
+            verify_key = VerifyKey(bytes.fromhex(verify_key))
+        super().__init__(verify_key=verify_key)
```

Comment:
Oh I thought these 2 do exactly the same thing? Genuine question coz I've been using validator for this. Am I missing anything?

Reply:
(I saw your commit reverting from validator back to this)

Reply:
I reverted it back for now because of its use in `convert_types` function. e.g. `convert_types(["id", "verify_key"], [UID, SyftVerifyKey])`. I will investigate if we can revert back to use validators